src/components/textarea.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(app) {
             }
             return 'formio/components/texteditor.html';
           }
+          else if (!$scope.readOnly && $scope.component.components) {
```

Comment:
Oh nice... I see what you are doing here. Allowing for a good editor to apply to the conditionals. Awesome! I would, however, prefer that the ace editor apply from a specific boolean that describes what it is doing. Even if it were a boolean called ```scripteditor``` attached to the textarea JSON would be more descriptive. Your thoughts?